pkgs/by-name/do/docling/package.nix

```diff
@@ -0,0 +1,75 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "docling";
+  version = "2.8.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "DS4SD";
+    repo = "docling";
+    rev = "v${version}";
+    hash = "sha256-XcrKufZWWE2b+INttHvehPvvQlHNWL/ghWysH/qmjw8=";
+  };
+
+  build-system = [
+    python3.pkgs.poetry-core
+  ];
+
+  dependencies = with python3.pkgs; [
+    beautifulsoup4
+    certifi
+    # deepsearch-glm # not yet packaged
+    docling-core
+    docling-ibm-models
+    docling-parse
+    easyocr
+    filetype
+    huggingface-hub
+    lxml
+    marko
+    onnxruntime
+    openpyxl
+    pandas
+    pyarrow
+    pydantic
+    pydantic-settings
+    # pypdfium2 # not yet packaged
+    python-docx
+    python-pptx
+    requests
+    rtree
+    scipy
+    typer
+  ];
+
+  optional-dependencies = with python3.pkgs; {
+    ocrmac = [
+      # ocrmac # not yet packaged
+    ];
+    rapidocr = [
+      onnxruntime
+      rapidocr-onnxruntime
+    ];
+    tesserocr = [
+      tesserocr
+    ];
+  };
+
+  pythonImportsCheck = [
+    "docling"
+  ];
+
```

Comment:
No tests ? If not, maybe it would be a good idea to add doChecks to false with a comment.

Reply:
I was waiting for this to build before doing the tests.
Because of tje two missing dependencies this wont work.
I checked yesterday, and it looks pdfium2 is packaged as part of a PR that was never merged, so we should be able to copy tyat.
However the other dependency is pulling different repos to compile them with cmake, its going to be a pain to actually package.
But once these two dependencies are in, yes, i should try to enable tje tests.